### DISCUSSION TITLE
小白的我还是想请教一下，如何开启ipv6？

### DISCUSSION THREAD

**Discussion by tobylibo** [coolsnowwolf/lede#10679] [Q&A / 快速问答]
**Lean大佬最新released的x86固件；
光猫桥接，openwrt软路由ppoe拨号成功后，可正常上网；**

我看之前有朋友提问过类似问题，关于禁用ipv6方法的，我学到了。我天真的以为只要逆向思维一下，就能开启ipv6，结果还是失败。

现在网络上关于开启ipv6的教程太乱了，鱼龙混杂，试了几个都不管用。

所以......还是恳请大佬出面，来一个官方点儿的开启ipv6的方法。多谢！

**Comment by JuanHpsy** [comment]
这是来自QQ邮箱的自动回复邮件。你好，这里是何娟，我已收到你的邮件，祝你天天有个好心情！

**Reply by tobylibo** [reply]
何娟你好，我是老王

**Comment by hanzeliu** [comment]
你是不是发错人了？我不是何娟

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: ***@***.***>
发送时间: 2022年12月29日 10:26
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [coolsnowwolf/lede] 小白的我还是想请教一下，如何开启ipv6？ (Discussion #10679)


何娟你好，我是老王

―

**Comment by rainchensh** [comment]
最新版的话，参考下面的说明吧
https://openwrt.org/docs/guide-user/network/ipv6/configuration

**Comment by JuanHpsy** [comment]
这是来自QQ邮箱的自动回复邮件。你好，这里是何娟，我已收到你的邮件，祝你天天有个好心情！

**Comment by fengqi** [comment]
extra里勾选ipv6helper，理论上编译出来的是可以直接用的，如果保留配置刷固件自己配不好，就不要勾选保留配置刷。
